include/vector3d.h

```diff
@@ -70,6 +70,11 @@ namespace core
 			return !operator==(other);
 		}
 
+        bool operator<(const vector3d<T>& other) const

```

Comment:
I removed this function for a reason!

Reply:
removed.

Reply:
I added it temporarily because I needed to perform core::max on this type just for when _mm_cmplt/gt are broken. See commit previous to last one, I replaced vector3simd for vector3d, somewhere in IImage I think

Reply:
fixing the compile issue now.

Reply:
changed to a macro in vectorSIMDf, works now